packages/storybook-html/src/components/card/card.docs.mdx

```diff
@@ -0,0 +1,178 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
+
+# Card
+
+<StatusBadge.Development />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
+
+The card component is used to provide a brief summary of content or a task, often with a link to more detail. Cards are frequently displayed alongside other cards to group related content or tasks.
+
+## Default example
+
+### Default card
+
+<Canvas withSource="none">
+  <Story id="components-cards--default-card" />
+</Canvas>
+
+```html
+<div class="card">
+  <div class="card__header"></div>
+  <div class="card__body">
+    <a class="card__link" href="#">
+      <div class="card__image-container" style="background-image: url('https://campuses.uq.edu.au/files/9749/gatton-card2.jpg');"></div>
+    </a>
+    <div class="card__content">
+      <h2 class="card__title">
+        <a class="card__link" href="#">Title of item</a>
+      </h2>
+      <div class="card__date" tabindex="0">Sunday 16 August, 2015</div>
+      <div class="card__location">
+        <a class="card__link" href="#">Gatton campus</a>
+      </div>
+      <div class="card__summary" tabindex="0">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
```

Comment:
I am not sure we need tabindex in these examples.... tabindex should be avoided at alll times as the nature of UQ sites means we do not control all content going in and therefore can not manage the whole page tabindex, better to let the content flow naturally.

Reply:
Done